binding.gyp

```diff
@@ -36,11 +39,14 @@
             'src/unix/base64.cc',
             'src/unix/kerberos_gss.cc',
             'src/unix/kerberos_unix.cc'
-          ],
+          ]
+        }],
+        ['(OS=="mac" or OS=="linux") and (kerberos_use_rtld!="true")', {
           'link_settings': {
             'libraries': [
               '-lkrb5',
-              '-lgssapi_krb5'
+              '-lgssapi_krb5',
+              '-lcom_err'
```

Comment:
> Is `com_err` new or pulled in differently somehow previously?

It’s the library that provides the `error_message` function, I think it was previously just omitted? Happened to work fine because krb5/gssapi_krb5 have transitive dependencies on it, so it would get loaded either way.

I’ll remove it again in the interest of making as little changes as possible to the regular addon build in this PR but I think it should be safe (and make sense) to include it in general, we already need to do this for the static library version (directly below this).

> This means it is a new statically linked library right?

It’s dynamically linked (unless you’re building the entire addon as a static library).

Reply:
Thanks for the explanation, makes sense, I'm not against keeping it here, but we can reintroduce it in another iteration